Apps/README.md

```diff
@@ -67,7 +67,7 @@ Both of these tools can't compare in terms of features and user interface with t
 ## Other
 
 - [CheatSheet](https://mediaatelier.com/CheatSheet/): Tap the command key for long to see all the keyboard shortcuts of the current app.
-- [Mou](http://25.io/mou/): Markdown editor for developers.
+- [Mou](https://apps.apple.com/app/mou-markdown-editor/id1308265497): Markdown editor for developers.
```

Comment:
This new link cannot be opened on Android, it just says connecting. The original link still works, maybe keep that?

Reply:
@hugovk It's a link to the Apple App Store so it makes sense that you can't open on Android :)

Reply:
Sure, but do we want people only to use this guide from a Mac?

I think it would still be useful for people to be able to find out more about these tools from any device (Android, Windows, whatever).

Reply:
> I think it would still be useful for people to be able to find out more about these tools from any device (Android, Windows, whatever).
I agree, new link should work for non-Apple devices. If you could confirm this is the case on Android?

Reply:
Perfect, thanks!